### PR TITLE
Implement keepalive support

### PR DESCRIPTION
**Pull request by tsipinakis**
## Please describe the change you are making

Teach containerssh about keepalive packets to avoid the logs being spammed by 'unknown global request logs'. Additionally, add settings for containerSSH to generate its own keepalives if necessary.

## Are you the owner of the code you are sending in, or do you have permission of the owner?

Sent with permission of the owner.

## The code will be published under the MIT-0 license. Have you read and understood this license?

Yes.

### CODE REVIEW

File: internal/sshserver/serverImpl.go [hunk 1]
```diff
@@ -510,6 +540,23 @@ func (s *serverImpl) handleGlobalRequests(
 		}
 		requestID := s.nextGlobalRequestID
 		s.nextGlobalRequestID++
+
+		if request.Type == "keepalive@openssh.com" {
```
Comment (janosdebugs) [inline internal/sshserver/serverImpl.go:544]: Please implement it as a switch-case and put the components into separate functions. This function will need to be extended in the future. It's better to not create the need for refactoring later if we can avoid it.

**Review by janosdebugs — CHANGES_REQUESTED** [review]
One minor change, otherwise looks good. Tests are needed though.

**Review by janosdebugs — DISMISSED** [review]
Looks good!

**Comment by tsipinakis** [comment]
Squashed & signed, this needs a re-approval to be merged.

**Review by janosdebugs — APPROVED** [review]

**Pull request merged**